rex/renewable_resource.py

```diff
@@ -500,15 +510,16 @@ def _get_calculated_ds(self, val, ds_name, var_name, ds_slice):
         out = linear_interp(ts1, v1, ts2, v2, val)
         return out
 
-    def _set_sam_res(self, values, dsets, SAM_res, time_slice, sites):
+    @staticmethod
+    def _set_sam_res(res, values, dsets, SAM_res, time_slice, sites):
```

Comment:
now that this is being touched again maybe we should use the opportunity to add to the doc string?

Reply:
yep good call!